www/storage.js

```diff
@@ -0,0 +1,69 @@
+// Based on code found at:
+// https://github.com/RinteRface/shinyLocalStorage/tree/main
```

Comment:
Please include the MIT licence requirements:
https://github.com/RinteRface/shinyLocalStorage/tree/main?tab=MIT-1-ov-file#readme

```
Copyright (c) 2024 David Granjon

Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
```

Reply:
Added